docs/index-ref.json

```
@@ -1,3 +1,3 @@
 {
-    "appref": "v1.10.34"
+    "appref": "v1.10.35"
```

```suggestion
    "appref": "v1.10.36"
```

bumping to 1.10.36 which picks up https://github.com/microsoft/pxt/pull/9106

---

Looks good